example/lambdas/python-processing/pyproject.toml

```diff
@@ -0,0 +1,34 @@
+[project]
+name = "cumulus-python-processing"
+version = "21.0.1"
+description = "Python reference activity that utilizes the Cumulus Process SDK."
+requires-python = "~=3.12.0"
+
+dependencies = [
+    "boto3~=1.40.29",
+    "cumulus-process==1.6.0",
+    "cumulus-message-adapter-python==2.4.0",
+    "six>=1.17.0",
+]
+
+[dependency-groups]
+dev = [
+    "mypy~=1.19.0",
+    "pytest~=9.0.2",
+    "pytest-mock~=3.15.1",
+    "ruff~=0.14.0",
+]
+
+[tool.ruff]
+extend = "../../../pyproject.toml"
+
+[tool.mypy]
+check_untyped_defs = true
+disallow_any_unimported = true
+follow_untyped_imports = true
+local_partial_types = true
+strict_bytes = true
+strict_equality = true
+warn_redundant_casts = true
+warn_unreachable = true
+warn_unused_ignores = true
```

Comment:
The same appears to be true for mypy as well: https://mypy.readthedocs.io/en/stable/config_file.html


```suggestion
```

Reply:
Fixed!